src/plusplus.coffee

```diff
@@ -142,7 +142,7 @@ module.exports = (robot) ->
     reasons = scoreKeeper.reasonsForUser(name)
 
     reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
-                     "#{name} has #{score} points. here are some raisins:" +
+                     "#{name} has #{score} points. here are some reasons:" +
```

Comment:
Please use topical pull requests and split the previous change (karma as alias) from this verbiage change.